packages/istanbul-reports/lib/text/index.js

```diff
@@ -87,46 +124,46 @@ function findNameWidth(node, context) {
     return last;
 }
 
-function makeLine(nameWidth) {
+function makeLine(nameWidth, missingWidth) {
     const name = padding(nameWidth, '-');
     const pct = padding(PCT_COLS, '-');
     const elements = [];
 
     elements.push(name);
     elements.push(pct);
+    elements.push(padding(PCT_COLS + 1, '-'));
     elements.push(pct);
     elements.push(pct);
-    elements.push(pct);
-    elements.push(padding(MISSING_COL, '-'));
-    return elements.join(COL_DELIM) + COL_DELIM;
+    elements.push(padding(missingWidth, '-'));
+    return elements.join(DELIM.replace(/ /g, '-'));
 }
 
-function tableHeader(maxNameCols) {
+function tableHeader(maxNameCols, missingWidth) {
     const elements = [];
     elements.push(formatName('File', maxNameCols, 0));
     elements.push(formatPct('% Stmts'));
-    elements.push(formatPct('% Branch'));
+    elements.push(formatPct('% Branch', PCT_COLS + 1));
     elements.push(formatPct('% Funcs'));
     elements.push(formatPct('% Lines'));
-    elements.push(formatPct('Uncovered Line #s', MISSING_COL));
-    return elements.join(' |') + ' |';
+    elements.push(formatPct('Uncovered Line #s', missingWidth));
+    return elements.join(DELIM);
 }
 
-function missingLines(node, colorizer) {
+function missingLines(node, colorizer, missingWidth) {
```

Comment:
Can't we just replace missingLines and missingBranches with `nodeMissing`?  I'd prefer we not duplicate logic.  Probably just have the caller to nodeMissing handle the colorizing / trimming.

Reply:
I didn't do it before to try to modify the less code possible for this pull-request, but I've reviewed it and you are right, code is duplicated, so I've unified all of them inside `nodeMissing()` function as you suggested.